client/src/components/admin/AdminDashboard.tsx

```
@@ -0,0 +1,123 @@
+            Support Requests
+          </button>
+        </nav>
+      </div>
+
+      {/* Mobile Menu Icon */}
+      <div className="fixed top-0 left-0 w-full bg-white shadow-md md:hidden z-30 flex items-center justify-between p-4">
+        <button
+          onClick={() => setSidebarOpen(!sidebarOpen)}
+          className="bg-purple-700 text-white p-2 rounded shadow"
+          title="Toggle Sidebar"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)